web/config/sync/user.role.moderator.yml

```diff
@@ -0,0 +1,14 @@
+uuid: d7b1ca0e-4271-4124-a560-9610e1a55679
+langcode: en
+status: true
+dependencies: {  }
+id: moderator
```

Comment:
Should we use the id `moderator` ? I think it would be a good idea to prefix our moderator/editor roles with "admin-" since the CommunityService also contains roles (and even a moderator role?)